src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
+.nav {
+  margin: 0;
+
+  &__links {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+
+    & li:last-child a {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $page-text-color;
+    height: 60px;
+    display: inline-block;
+    line-height: 60px;
+    margin-right: 20px;
+    position: relative;
+    font-weight: 500;
+
+    &:hover {
+      color: $button-color;
+    }
+  }
+}
+
+.is-active {
+  color: $button-color;
+}
+
+.is-active::after {
+  content: ' ';
+  height: 4px;
+  width: 100%;
+  background-color: $button-color;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
```

Comment:
The '.is-active' class and its pseudo-element are part of the 'nav__link' block, so they should be nested inside it. Also, remember to use '&' to refer to the parent selector when defining pseudo-elements or modifiers.